CaImAn/caiman_run.py

```diff
@@ -151,6 +158,9 @@
         except TypeError:
             utils.print_to_intercept(PARAM_WRONG_TYPE)
             sys.exit()
+        except Exception as error:
+            utils.print_error(error, MOTION_CORREC)
```

Comment:
Can we change Motion correction text from capital letters?

Reply:
yeap!